test/test_docker_pull.py

```diff
@@ -11,23 +11,23 @@ class DockerPullTest(unittest.TestCase):
     def setUp(self):
         print(f"SetUp {self.__class__.__name__}")
         self.test_environment = exaslct_utils.ExaslctTestEnvironmentWithCleanUp(self, exaslct_utils.EXASLCT_DEFAULT_BIN)
-        self.registry_container, self.registry_host, self.registry_port = self.test_environment.create_registry()
-        self.test_environment.clean_images()
```

Comment:
This clean you should keep, to cleanup potential still existing images in the cache of the docker service

Reply:
The clean is still there, check line 18